lib/service/service.go

```diff
@@ -2066,6 +2068,26 @@ func (process *TeleportProcess) initSSH() error {
 	return nil
 }
 
+func compareVersions(ctx context.Context, c *Connector) error {
+	resp, err := c.Client.Ping(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	serverVersion, err := semver.NewVersion(resp.ServerVersion)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	teleportVersion, err := semver.NewVersion(teleport.Version)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if math.Abs(float64(serverVersion.Major-teleportVersion.Major)) > 1 {
+		logrus.Warnf("Only two major teleport versions are supported. "+
```

Comment:
Comments says "Only two major teleport versions are .." but condition allows only for 1 major version difference. 

Also 
> "Only two major teleport versions are supported

and

>  ... a supported version ...

sentences are configuring for me a bit. For instance what is "supported version" ? 

Maybe something: "Only two major versions difference  between auth client and auth server are supported"